test/Helper/GravatarTest.php

```diff
@@ -302,4 +302,12 @@ public function testSetAttribsDocCommentHasDeprecated()
 
         $this->assertContains('@deprecated', $comment);
     }
+
+    /**
+     * @expectedException PHPUnit\Framework\Error\Deprecated
```

Comment:
Please look at Travis:

> 2) ZendTest\View\Helper\GravatarTest::testGetAttribsIsDeprecated
ReflectionException: Class PHPUnit\Framework\Error\Deprecated does not exist

https://travis-ci.org/zendframework/zend-view/jobs/275727410#L733